zwave_js_server/model/device_class.py

```diff
@@ -7,7 +7,7 @@
 from typing import List, Optional, TypedDict
 
 
-class DeviceClassDataType(TypedDict):
+class DeviceClassDataType(TypedDict, total=False):
```

Comment:
Where is `total` being used?

Reply:
mypy didn't like the empty dict coming in...
if the node is being added this info is added at a later stage

Reply:
`total` is part of the TypedDict api in standard library.